app/templates/components/ui-table/cell/events/view/speakers/cell-buttons.hbs

```diff
@@ -1,17 +1,13 @@
 {{this.record}}
-<div class="hidden ui divider"></div>
+<br><br>
 <div class="ui horizontal compact basic buttons">
-  <LinkTo @route="events.view.speakers.edit" @model={{this.extraRecords.id}}>
-    <UiPopup @content={{t "View"}} @class="ui icon button" @click={{action this.props.actions.viewSpeaker this.extraRecords.id}} @position="left center">
-      <i class="unhide icon"></i>
-    </UiPopup>
-  </LinkTo>
-  <LinkTo @route="events.view.speakers.edit" @model={{this.extraRecords.id}}>
-    <UiPopup @content={{t "Edit"}} @class="ui icon button" @click={{action this.props.actions.editSpeaker this.extraRecords.id}} @position="left center">
-      <i class="edit icon"></i>
-    </UiPopup>
-  </LinkTo>
+  <UiPopup @content={{t "View"}} @class="ui icon button" @click={{action this.props.actions.viewSpeaker this.extraRecords.id this.extraRecords.event.id}} @position="left center">
```

Comment:
You can just pass this.extraRecords. That's the speaker. And then access speaker.event.id

Why making it so complicated?

Reply:
Yes we can pass just one argument instead of two.

Reply:
Done